lib/feedback.ml

```diff
@@ -42,7 +42,7 @@ type feedback_content =
   (* Extra metadata *)
   | Custom of Loc.t option * string * xml
   (* Generic messages *)
-  | Message of level * Loc.t option * Pp.t
+  | Message of level * Loc.t option * Quickfix.t list * Pp.t
```

Comment:
This new type for `Message` seems strange, maybe it would best to introduce a new type of message `Quickfix`.

Reply:
For instance what are clients supposed to do when a Debug message with a quickfix payload arrives?

Reply:
In the beginning it was just attached to Warning, then also Error and now to Message, I think @SkySkimmer suggested it. It surely makes the code simpler when a warning becomes and error. I'm also fine adding a Quickfix message, but that would then be unrelated to any other message, and that is bad, since a fix proposal with no other message attached is going to be very surprising for the user.

I think the client should handle quickfixes the same way. I don't have a use case for a Debug message suggesting a code change, but if you receive one just display it together with the proposed fix if any.